tests/assets/workflow_example/daylightfactor.yaml

```diff
@@ -64,104 +82,123 @@ templates:
       other by a space.
     operator: radiance-operator
     inputs:
-      parameters:
-        - name: input-files
-          description: Octree input files separated by space. 
-        - name: output-file
-          description: Path to output octree file.
-    command: oconv -f "{{inputs.parameters.input-files}}" > "{{inputs.parameters.output-file}}" 
+      artifacts:
+        - name: sky-file
+          location: project-folder
+          path: sky.file # Note that this is overwritten in the DAG by the output artifact path from the generate-sky task template
+          task_path: sky.file
+        - name: scene-files
+          location: project-folder
+          path: model
+          task_path: '.'
+    command: |
+      oconv \
+        ./static/aperture/*.mat ./static/aperture/*.rad \
```

Comment:
You can't pass * to a radiance command. What didn't you like about the initial set-up?

Reply:
Didn't really have any qualms with the initial setup, although it wasn't super clear what the artifacts were (were you passing a folder or a file?).

With regards to this command, I have no idea what radiance can/cannot accept, I just blindly copy/pasted this: https://github.com/ladybug-tools/radiance-folder-structure/tree/master/project_folder/model#ray-tracing-with-rtrace--rcontrib--rpict

Reply:
Resolved it here: d3b8a55 

Feel free to re-open if not happy with it.